src/python/pants/core_tasks/register.py

```diff
@@ -59,7 +59,7 @@ def register_goals():
   # Pantsd.
   kill_pantsd = task(name='kill-pantsd', action=PantsDaemonKill)
   kill_pantsd.install()
-  kill_pantsd.install('clean-all')
+  kill_pantsd.install('clean-all', first=True)
```

Comment:
Can you comment here as to why this is necessary to run first?

Reply:
done